e2e/sidecars/sidecars_test.go

```diff
@@ -49,14 +50,18 @@ sidecars:
   nginx:
     port: 80
     image: %s    # Image URL for sidecar container.
+    variables:
+      NGINX_PORT: %s
```

Comment:
Is it possible to set `variables` or `secrets` per-environment, i.e. `dev` and `prod`

Reply:
Yeah something like this

```
# Optional. You can override any of the values defined above by environment.
environments:
  test:
    variables:
      NGINX_PORT: 80
```

Reply:
Yeah I think you can do the override like this

```
sidecars:
  nginx:
    port: 80
    image: %s    # Image URL for sidecar container.
    variables:
      NGINX_PORT: 80

environments:
  dev:
    nginx:
      variables:
        NGINX_PORT: 8080
  prod:
    nginx:
      variables:
        NGINX_PORT: 5000
```